docs/src/details/integration/chip/arm.rst

```diff
@@ -52,6 +52,13 @@ To add Arm2D to your project, follow these steps:
 Neon Acceleration
 -----------------
 
-Several Cortex-A microprocessors support the `Neon SIMD <https://www.arm.com/technologies/neon>`__ instruction set. LVGL has built-in support to improve the performance of software rendering by utilizing Neon instructions. To enable Neon acceleration, set ``LV_USE_DRAW_SW_ASM`` to ``LV_DRAW_SW_ASM_NEON`` in ``lv_conf.h``.
+Some ARM processors with the ARMv7 architecture and every ARM processor from the ARMv8 architecture support the `Neon SIMD <https://www.arm.com/technologies/neon>`__ instruction set.
+LVGL has built-in support to improve the performance of software rendering by utilizing Neon instructions.
 
+Architecture Support
+--------------------
+
+Both 32-bit and 64-bit ARM architectures are supported. Simply set ``LV_USE_DRAW_SW_ASM`` to ``LV_DRAW_SW_ASM_NEON`` in ``lv_conf``.
 
+.. note::
+   All ARM64 processors include Neon support as a mandatory feature starting with the ARMv8 architecture specification. This makes Neon acceleration universally available on all 64-bit ARM platforms, including current and future ARM architectures.
```

Comment:
Here also, even though all ARM64 may have Neon support, they are all ARMv8 ISA on A profile, M profiles are exceptions to this rule, I think it is good to make this point clear because LVGL is widely deployed on microcontrollers which are ARMv7/8 ISA but on M profiles that does not support Neon.